File: ser/ser.go
package main

import "fmt"

func calculateSER(strikeouts int, inningsPitched float64) float64 {
    // SER = (strikeouts / inningsPitched) * 9
    return (float64(strikeouts) / inningsPitched) * 9
}

func main() {
    strikeouts := 150
    inningsPitched := 180.0

    SER := calculateSER(strikeouts, inningsPitched)
    fmt.Printf("Strikeout Efficiency Rating (SER): %.2f\n", SER)
}

File: ser/ser.py
def calculate_SER(strikeouts, innings_pitched):
    """
    Calculate Strikeout Efficiency Rating (SER) for a pitcher.
    
    SER = (strikeouts / innings_pitched) * 9
    
    Args:
    strikeouts (int): Number of strikeouts by the pitcher.
    innings_pitched (float): Number of innings pitched by the pitcher.
    
    Returns:
    float: Strikeout Efficiency Rating (SER).
    """
    return (strikeouts / innings_pitched) * 9

